rllib/algorithms/dqn/tests/test_repro_dqn.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+
+import os
+import ray
+from ray.tune import register_env
+import ray.rllib.algorithms.dqn as dqn
+from ray.rllib.examples.env.deterministic_envs import DeterministicCartPole
+from ray.rllib.utils.test_utils import check_reproducibilty
+
+
+class TestReproPPO(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        ray.init(local_mode=True)
+
+    @classmethod
+    def tearDownClass(cls):
+        ray.shutdown()
+
+    def test_reproducibility_dqn_cartpole(self):
+        """Tests whether the algorithm is reproducible within 3 iterations
+        on discrete env cartpole."""
+
+        register_env(
```

Comment:
in the future instead of registering, pass a lambda that takes as input a config, or pass an env whose constructor is a cfg. then pass an env config

Reply:
sg.